ext/nnstreamer/tensor_decoder/meson.build

```diff
@@ -66,3 +66,26 @@ static_library('nnstreamer_decoder_bounding_boxes',
   install: true,
   install_dir: nnstreamer_libdir
 )
+
+# pose estimation
+decoder_sub_pose_estimation_sources = [
+  'tensordec-pose.c'
+]
+
+nnstreamer_decoder_pose_estimation_sources = []
+foreach s : decoder_sub_pose_estimation_sources
+  nnstreamer_decoder_pose_estimation_sources += join_paths(meson.current_source_dir(), s)
+endforeach
+
+shared_library('nnstreamer_decoder_pose_estimation',
+  nnstreamer_decoder_pose_estimation_sources,
+  dependencies: [nnstreamer_dep, glib_dep, gst_dep, libm_dep],
```

Comment:
remove dependency math lib, it is unnecessary.
```
libm_dep
```

Reply:
Done.